bluebutton-data-model-codegen/src/main/java/gov/hhs/cms/bluebutton/data/model/codegen/RifLayoutsProcessor.java

```diff
@@ -549,6 +556,36 @@ private TypeSpec generateHeaderEntity(MappingSpec mappingSpec) throws IOExceptio
 			headerEntityClass.addMethod(childGetter);
 		}
 
+		// Add the parent-to-child join field and accessor for an inner join
+		// relationship
+		if (mappingSpec.getHasInnerJoinRelationship()) {
```

Comment:
Can you update your PR description to include the auto-generated `Beneficiary`, and the two auto-generated History tables? Too early on a Monday to trust myself that this code is generating the right annotations.

Reply:
I know you want to see the files for these classes, I'm just not sure from your comment where or how to add them..

Reply:
New fields in `Beneficiary`:

``` 
  @OneToMany(
      mappedBy = "beneficiaryId",
      orphanRemoval = false,
      cascade = CascadeType.ALL
  )
  @LazyCollection(LazyCollectionOption.FALSE)
  @OrderBy("beneficiaryId ASC")
  private List<BeneficiaryHistory> beneficiaryHistories = new LinkedList<>();

  @OneToMany(
      mappedBy = "beneficiaryId",
      orphanRemoval = false,
      cascade = CascadeType.ALL
  )
  @LazyCollection(LazyCollectionOption.FALSE)
  @OrderBy("beneficiaryId ASC")
  private List<MedicareBeneficiaryIdHistory> medicareBeneficiaryIdHistories = new LinkedList<>();
```

`BeneficiaryHistory` and `MedicareBeneficiaryIdHistory` do not have any "new" fields, but `MedicareBeneficiaryIdHistory.beneficiaryId` has been updated to be a varchar(15). That work was done as part of a separate ticket though, https://jira.cms.gov/browse/BLUEBUTTON-1010.